examples/patterns/navigation/default.html

```diff
@@ -4,7 +4,7 @@
 category: _patterns
 ---
 
-<header id="navigation" class="p-navigation--light">
+<header id="navigation" class="p-navigation">
```

Comment:
If this is the only example of the pattern please remove the navigation directory and rename this file `navigation.md`

Reply:
As above, two patterns - default and sidebar?